core/core.go

```diff
@@ -0,0 +1,2 @@
+// Package core contains the most-common System implementations according to the engo.io/ecs definition.
```

Comment:
Not sure if this is what you were referring to?

From https://blog.golang.org/godoc-documenting-go-code

```go
// Package sort provides primitives for sorting slices and user-defined
// collections.
package sort
```